src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,61 @@
+package nextstep.courses.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Session {
+
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
```

Comment:
```suggestion
    private LocalDateTime startDate;
    private LocalDateTime endDate;
```

하나의 객체로 도출 해볼수 있지 않을까요?